kura/org.eclipse.kura.ai.triton.server/OSGI-INF/metatype/org.eclipse.kura.ai.triton.server.TritonServerContainerService.xml

```diff
@@ -158,6 +158,33 @@
             default="4194304"
             min="1">
         </AD>
+        
+        <AD id="enable.metrics"
+            name="Enable Triton Server Metrics"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="true"
+            description="Enable the Triton Server Metrics feature. This property enables the default, CPU and GPU metrics, if available.">
```

Comment:
```suggestion
            description="Enable the Triton Server Metrics feature. This property enables the default CPU and GPU metrics, if available.">
```

Reply:
@MMaiero The property enables the metrics feature that contains CPU, GPU and standard statistics. I'll rephrase in this way:

> Enable the Triton Server Metrics feature. This property enables the standard statistics and CPU/GPU metrics, if available.